awl/mslider.cpp

```diff
@@ -229,20 +229,21 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
       //    draw tick marks
       //---------------------------------------------------
 
-  	QFont f(p.font());
-   	f.setPointSize(6);
-   	p.setFont(f);
+  	  QFont f(p.font());
+   	  f.setPointSize(6);
+   	  p.setFont(f);
       p.setPen(QPen(Qt::darkGray, 2));
-   	QFontMetrics fm(f);
+   	  QFontMetrics fm(f);
       int xt = 20 - fm.width("00") - 5;
```

Comment:
Since we are you using negative numbers, we need to adjust xt too...  This fm.width("00") should be changed to fm.width("-00") and I've verified that will allow for enough space between the numbers and the red-green rectangle.

![screenshot 188](https://cloud.githubusercontent.com/assets/6502474/24065732/ae4f0804-0b43-11e7-80cd-57c345ea9e03.png)